pages/linux/dmesg.md

```diff
@@ -0,0 +1,15 @@
+# dmesg
+
+> Write the kernel messages to standard output.
```

Comment:
This feels a teensy bit awkward, but it's nothing to worry about - I can't think of anything better :P